parser/csv_tree_parser_test.go

```diff
@@ -139,6 +157,8 @@ func TestMakeTree(t *testing.T) {
 				}
 			}
 
+			treemap.SetNamesFromPaths(tree)
```

Comment:
ok, so you added Name field to test cases because of this call I gueess

Reply:
> hm, something does not sit right with so many inlined calls spread across codebase for strings replacement. that will not scale well.

Maybe another solution could be to just leave it the entity everywhere until SVG rendering. I think there was a problem with that, as it was my first idea. Nor sure, will give it another try.

Reply:
Ok, it was text size calculations that were the reason.
#27 Solves that by a single replacement when constructing the UIText.